src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CapabilitiesManager.cs

```diff
@@ -80,15 +81,15 @@ private async Task<string> ComputeRootPathAsync()
 
         if (initializeParams.WorkspaceFolders is [var firstFolder, ..])
         {
-            return firstFolder.Uri.GetAbsoluteOrUNCPath();
+            return firstFolder.DocumentUri.GetRequiredParsedUri().GetAbsoluteOrUNCPath();
```

Comment:
>GetAbsoluteOrUNCPath [](http://example.com/codeflow?start=66&length=20)

Any place that required this now calls it via the result of DocumentUri.GetRequiredParsedUri